### ISSUE TITLE
The countdown timer is only counting down from 10:00, after I selected to change the prep time to 15 min. for the next hour

### ISSUE THREAD

**Issue by shaneliammurphy** [Boostly/fulfillment-client#6]
![image](https://user-images.githubusercontent.com/34665139/34216768-88ea0e30-e566-11e7-96dc-cc937eada776.png)

**Comment by mikeyamadeo** [comment]
@shaneliammurphy changing prep time only effects future orders

**Comment by shaneliammurphy** [comment]
@mikeyamadeo Correct. I submitted a change for future orders to be 15 minutes for the next hour and did some new orders, and the timer for the new orders was still at 10:00.

**Comment by mikeyamadeo** [comment]
@shaneliammurphy gotcha. Which makes sense because this is hard coded god mode :). Maybe code this up for the demo tho?

**Comment by shaneliammurphy** [comment]
Yeah, coding for the demo might be good. However, we can just verbally talk
about it if necessary, and we can get by. So, if your time is better spent
doing other projects that will help us launch on time, I would prioritize
those. I'll leave that in your court to decide.

Thanks,

*Shane Murphy*
CEO | Boostly
(269) 569-4562
